extensions.json

```
@@ -5,6 +5,11 @@
       "download": "https://github.com/vlang/vscode-vlang/releases/download/0.1.12/vscode-vlang-0.1.2.vsix",
       "version": "0.1.2"
     },
+    {
+      "id": "42Crunch.vscode-openapi",
+      "repository": "https://github.com/42Crunch/vscode-openapi.git",
+      "version": "3.8.0"
```

Similar remark as in the other PR, but this time there is a dedicated release tag that we can checkout. 👍 

```suggestion
      "repository": "https://github.com/42Crunch/vscode-openapi",
      "version": "3.8.0",
      "checkout": "v3.8.0"
```